### PR TITLE
Add uninstall profile

### PR DESCRIPTION
**Pull request by hvelarde**

### CODE REVIEW

**Comment by hvelarde** [comment]
@plone/framework-team I need a review please; if no issues are found please merge so I can ask for a new release.

File: buildout.cfg [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [buildout]
-extends = test-4.3.x.cfg
+extends = test-5.0.x.cfg
```
Comment (rodfersou) [inline buildout.cfg]: ops.. tenho que tirar isso..

Comment (jensens) [reply buildout.cfg]: Why 5.0 not 5.1?

Comment (jensens) [reply buildout.cfg]: ah, sorry, now I see, its for 1.x!

Comment (rodfersou) [reply buildout.cfg]: @hvelarde my mistake.. I thought it was one of my PR in another project.

**Comment by jensens** [comment]
released. 1.0.1 was brown bag b/c of a local configuration problem, so I did a 1.0.2 which is fine.

**Comment by jensens** [comment]
@hvelarde I think we need it on master too, right?

**Comment by hvelarde** [comment]
yes, but I might be different; let me see if I can push it later today.

**Pull request merged**